commands/mr/create/mr_create.go

```diff
@@ -501,7 +503,16 @@ func mrBodyAndTitle(opts *CreateOpts) error {
 		if opts.Description == "" {
 			var body strings.Builder
 			for i := len(commits) - 1; i >= 0; i-- {
-				fmt.Fprintf(&body, "- %s\n", commits[i].Title)
+				// adds 2 spaces for markdown line wrapping
+				fmt.Fprintf(&body, "- %s  \n", commits[i].Title)
```

Comment:
Oops this rules out my comment here https://github.com/profclems/glab/pull/819#issuecomment-897382936.

Did you test with the this after adding the spaces?

Reply:
Ah yes, the screenshot is slightly outdated I think. The two spaces aligns the main body of the commit but the `closes  #902` is then misaligned. One option would be to replace all `\n` with double space in the body?

Reply:
I think that happens because there is `\n\n` before the `closes #902`. Maybe we can replace the `\n\n` with `\n`??

Reply:
Another option is to insert double spaces before each `\n`. So we replace all `\n` with `<double-space>\n`

Reply:
Went with a combination of both, replacing `\n\n` with `<double-space>\n`. Looks much better now, thanks for the suggestions

![2021-08-12_101433](https://user-images.githubusercontent.com/1794071/129171567-0f04f283-8b9d-4e44-9b59-6958ac1475f0.png)